students/yixingxu/lesson09/mailroom5.py

```diff
@@ -0,0 +1,162 @@
+import sys
+import os
+
+class Donor():
+    def __init__(self, name):
+        self._name = name
+        self._donations = []
+
+    @property
+    def name(self):
+        return self._name
+
+    @name.setter
+    def name(self, donor_name):
+        self._name = donor_name
```

Comment:
You don't need to use a getter and setter if you aren't adding any logic or processing beyond setting the attribute